var/spack/repos/builtin/packages/rdma-core/package.py

```diff
@@ -31,11 +31,13 @@ class RdmaCore(CMakePackage):
     homepage = "https://github.com/linux-rdma/rdma-core"
     url      = "https://github.com/linux-rdma/rdma-core/releases/download/v17.1/rdma-core-17.1.tar.gz"
 
-    version('17.1', '1d19caf554f815990af5c21356ac4d3a')
-    version('13', '6b072b4307d1cfe45eba4373f68e2927')
+    version('20', sha256='bc846989f807cd2b03643927d2b99fbf6f849cb1e766ab49bc9e81ce769d5421')
+    version('17.1', sha256='7f5ca9715dce922404851f6ca648399d3dfc3fc2789296f27e94bd2299c68aff')
+    version('13', sha256='e5230fd7cda610753ad1252b40a28b1e9cf836423a10d8c2525b081527760d97')
 
     depends_on('libnl')
     conflicts('platform=darwin', msg='rdma-core requires FreeBSD or Linux')
+    conflicts('%intel', msg='rdma-core can not be build with intel (use gcc)')
```

Comment:
Typo: build -> built

I think "cannot" is more common than "can not" as well. I would also add "instead" after "use gcc".